afs/afs-storage-api/src/main/java/com/powsybl/afs/storage/events/TimeSeriesAllRemoved.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2018, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.storage.events;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class TimeSeriesAllRemoved extends NodeEvent {
```

Comment:
Rename to TimeSeriesCleared ?

Reply:
This event has to be consistent with AppStorage.removeAllTimeSeries

Reply:
Ok renamed to TimeSeriesCleared